internal/bminventory/inventory_test.go

```diff
@@ -6446,6 +6446,36 @@ var _ = Describe("[V2ClusterUpdate] cluster", func() {
 					verifyApiErrorString(reply, http.StatusBadRequest,
 						"Setting Machine network CIDR is forbidden when cluster is not in vip-dhcp-allocation mode")
 				})
+				It("Machine network CIDR in non dhcp for dual-stack", func() {
```

Comment:
It would be nice to add tests for the non-happy path

Reply:
Sure, added a test with IPv6 network before IPv4